support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.jsx

```diff
@@ -33,6 +33,10 @@ function TermsPrivacy(props: PropTypes) {
     AUD: 185,
     CAD: 167,
     NZD: 200,
+    SEK: 0,
```

Comment:
👍 because we're using `fromCountryGroupId` below, these will never be used

Reply:
should we have a comment to that effect? Or maybe they can be removed?

Reply:
We have to make flow happy somehow...
The alternative is to leave them out, and handle the case where `regionalAmounts[currency]` doesn't exist.
How about turning `patronText` into a function to handle this, e.g.:
```
const getRegionalAmountString = (): ?string => ...
const patronText = () => {
  const amount = getRegionalAmountString();
  if (amount) {
    return (<div...
  }
  return null;
}
```

Reply:
This should never happen of course, but it's nice to defend against bugs creeping in later

Reply:
@TomPretty @tomrf1 @Aracho1 what do you guys think of the change I've made to address this?

Reply:
I'm marking this as resolved now ✅